lib/index.js

```diff
@@ -33,6 +33,8 @@ const rules = {
     'no-unsupported-ssr-properties': require('./rules/no-unsupported-ssr-properties'),
     'no-node-env-in-ssr': require('./rules/no-node-env-in-ssr'),
     'no-host-mutation-in-connected-callback': require('./rules/no-host-mutation-in-connected-callback'),
+    'no-static-imports-of-user-specific-scoped-modules-in-ssrable-components': require('./rules/no-static-imports-of-user-specific-scoped-modules-in-ssrable-components'),
+    'no-form-factor-in-ssrable-components': require('./rules/no-form-factor-in-ssrable-components'),
```

Comment:
```suggestion
    'ssr/no-static-imports-of-user-specific-scoped-modules': require('./rules/no-static-imports-of-user-specific-scoped-modules-in-ssrable-components'),
    'ssr/no-form-factor': require('./rules/no-form-factor-in-ssrable-components'),
```

Reply:
I don't know if you also want to change the file paths for the rules and READMEs to match the rule format:


Feedback from @wjhsf 

In the eslint plugin, there’s currently no consistency in the SSR rule names:
- `lwc/no-restricted-browser-globals-during-ssr`
- `lwc/no-unsupported-ssr-properties`
- `lwc/no-node-env-in-ssr`
- `lwc/no-unsuppoted-node-api-in-ssrable-components`
- `lwc/no-unguarded-async-event-in-ssr`
- `lwc/no-form-factor-in-ssrable-components`
- `lwc/no-static-imports-of-user-specific-scoped-modules-in-ssrable-components`

To have a more consistent naming convention (and terser names), please change all of them to the format `lwc/ssr/*`.

For the few older ones that have already been published, we could either do a rename and release a major, or we could keep the old ones as aliases, but mark them as deprecated.

Reply:
done